.buildbot.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Sorry, I just noticed these two. Unless we really really really need `bash`, we should do `#! /bin/sh` here. Also we shouldn't include your vim settings on the line below. Sorry for not spotting these earlier!

Reply:
Currently /bin/sh is linked to `dash`  @ltratt .   Should the rest of the script also be changed to work with dash ? or should be change the default shell in the container to `/bin/bash` ?

Reply:
Ideally the script should work with any `sh` descendent and shouldn't rely on bash or dash (or zsh or ...): I doubt there's anything in here which cares if it's bash or dash.

Reply:
Regarding the vim settings. This is what I get for copy pasting code from other repos. Will remove!